spec/requests/portfolio_items_rbac_spec.rb

```diff
@@ -10,6 +10,7 @@
   describe "GET /portfolio_items" do
     it 'returns status code 200' do
       allow(ManageIQ::API::Common::RBAC::Access).to receive(:new).with('portfolio_items', 'read').and_return(access_obj)
+      allow(ManageIQ::API::Common::RBAC::Roles).to receive(:assigned_role?).with(catalog_admin_role).and_return(false)
```

Comment:
These allows should be changed over to webmock calls, it will bring these specs inline with what @eclarizio has spent a lot of time on and allows us to test the full end to end calls without mocking responses.

Reply:
@syncrou I am not sure if we have to go down to the web mock layer, have a URL and all the stuff, define a json payload that seems an overkill. We should be able to stub out that specific method and have it return a pre-canned-response. The objects have already been converted to the proper instance type. If you think of layers
api -> rest -> url (the web mock stubs it at the URL layer) 3 layers down
With the current stubbing you are stubbing it directly at the API layer we don't have to go three levels down.

Reply:
I disagree about the 3 levels down, I like web_mock because it lets any request based call to run the calls out to the other microservices so we don't ignore testing that critical path.

I also see that with RBAC abstracted to the common gem those web_mock calls should be made in that gem and for RBAC in catalog we're not testing the calls, just the responses.
So, I'm okay with this as is.